### PR TITLE
Update "build" terminology for API v1

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Updates "build" references to "job".

# Reasons
The "build" terminology is out of date.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: src-api/source/includes/_builds_and_jobs.md [hunk 1]
```diff
@@ -165,26 +165,26 @@ curl -X POST https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/
...
     "build_num" : 25
   }
 }
 ```
 
 
-**`POST` Request:** Cancels the build and then returns a summary of the build.
+**`POST` Request:** Cancels the job and then returns a summary of the job.
 
 ## Trigger a new Job
```
Comment (rosieyohannan) [inline src-api/source/includes/_builds_and_jobs.md:189]: I just tried this and it only works if you have a job called "build" and it only triggers the job "build"

File: src-api/source/includes/_builds_and_jobs.md [hunk 2]
```diff
@@ -376,11 +376,11 @@ For more about build parameters, refer to the guide for <a href="https://circlec
 **Parameter** | **Description**
 ------- | -------------
 revision | The specific revision to build. Default is null and the head of the branch is used. Cannot be used with tag parameter.
-build_parameters | Additional environment variables to inject into the build environment. A map of names to values.
+build_parameters | Additional environment variables to inject into the job environment. A map of names to values.
 
 **Note** Triggering a new job with a branch is not currently supported with configurations that specify `version: 2.1`.
 
-## Trigger a new Build by Project (preview)
+## Trigger a new Job by Project
```
Comment (rosieyohannan) [inline src-api/source/includes/_builds_and_jobs.md:383]: This one seems to trigger the full pipeline - all workflows

File: src-api/source/includes/_builds_and_jobs.md [hunk 3]
```diff
@@ -393,7 +393,7 @@ curl -X POST https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/
   }
 ```
 
-**`POST` Request:** Triggers a build of the specified project, by branch, revision, or tag. Workflows will be run or scheduled in the same way as when a webhook from source control is received.
+**`POST` Request:** Triggers a job of the specified project, by branch, revision, or tag. Workflows will be run or scheduled in the same way as when a webhook from source control is received.
 
 
 <aside class="notice">
```
Comment (rosieyohannan) [inline src-api/source/includes/_builds_and_jobs.md:399]: This prerequ is outdated, pipelines are enabled by default

**Review by rosieyohannan — COMMENTED** [review]
Just a few comments, this terminology is difficult! I wonder if "job run" would be clearer in a few cases

File: src-api/source/includes/_artifacts.md [hunk 4]
```diff
@@ -63,17 +63,17 @@ curl https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/latest/a
 } ]
 ```
 
-Returns an array of artifacts produced by the latest build on a given branch.
+Returns an array of artifacts produced by the latest job on a given branch.
```
[the remark this answers is not in the document]
Comment (kbatuigas) [reply src-api/source/includes/_artifacts.md:66]: It seems to me that across the docs and even at CCI we use "job" to refer to both the definition as well as the run/execution... Let's give "job run" a try though!

File: src-api/source/includes/_builds_and_jobs.md (hunk 1 above)
Comment (kbatuigas) [reply src-api/source/includes/_builds_and_jobs.md:189]: @rosieyohannan Oof, thank you for catching that; I tried it as well and got the same result. I'll check internally

**Review by rosieyohannan — APPROVED** [review]
Looks great!

**Pull request merged**